### PR TITLE
New UI for GeoMol sections

### PR DESCRIPTION
**Pull request by vladyslav-tk**

### CODE REVIEW

**Comment by gberaudo** [comment]
@vladyslav-tk , some questions:
image

- should it still be possible to add new geometries?
- why are some point/line/rectangles de-activated?

**Comment by vladyslav-tk** [comment]
@gberaudo 

> should it still be possible to add new geometries?
no, it possible only from the draw panel

> why are some point/line/rectangles de-activated?
because they placed outside the area. Elisabeth asked to do like this

File: src/toolbox/ngm-gst-interaction.ts [hunk 1]
```diff
@@ -170,52 +159,72 @@ export class NgmGstInteraction extends LitElementI18n {
...
     this.viewer!.scene.requestRender();
   }
 
-  render() {
+  onGeomClick(geom: NgmGeometry) {
+    this.selectedId = this.selectedId === geom.id ? undefined : geom.id;
+    ToolboxStore.nextGeometryAction({id: geom.id, action: 'pick'});
+  }
+
+  disabledCallback(geom) {
```
Comment (gberaudo) [inline src/toolbox/ngm-gst-interaction.ts:177]: Rename to `geometryInsideExtent`

File: src/toolbox/ngm-gst-interaction.ts [hunk 2]
```diff
@@ -170,52 +159,72 @@ export class NgmGstInteraction extends LitElementI18n {
...
-          <div class="text"></div>
-          <i class="dropdown icon"></i>
+        <div class="ngm-section-format-container">
+          <label>${i18next.t('tbx_cross_sections_format_label')}</label>
+          <div class="ui fluid dropdown ngm-section-format">
+            <div class="text"></div>
+            <i class="dropdown icon"></i>
+          </div>
         </div>
+        <button class="ui button ngm-action-btn ${this.hasValidParams(geom) ? '' : 'disabled'}"
```
Comment (gberaudo) [inline src/toolbox/ngm-gst-interaction.ts:209]: Use the classMap directive.

**Review by gberaudo — APPROVED** [review]

**Pull request merged**